src/utils/graph_building.jl

```diff
@@ -7,21 +7,23 @@ export inverse_square, exp_decay
 using PyCall
 using ChemistryFeaturization
 using Serialization
+using Xtals
+using NearestNeighbors
+rc[:paths][:crystals] = @__DIR__ # because Xtals has a silly default thing
 
 # options for decay of bond weights with distance...
 # user can of course write their own as well
 inverse_square(x) = x^-2.0
 exp_decay(x) = exp(-x)
 
 """
-Function to build graph from a file storing a crystal structure (currently supports anything ase.io.read can read in). Returns an AtomGraph object.
+Build graph from a file storing a crystal structure (currently supports anything ase.io.read can read in). Returns an AtomGraph object.
```

Comment:
`build_graph` by itself doesn't return an `AtomGraph`, right?
```suggestion
Build graph from a file storing a crystal structure (currently supports anything Xtals.jl can read in). Returns the weight matrix and elements used for constructing an `AtomGraph`.
```